apps/bfd-model/bfd-model-rif-samples/src/main/java/gov/cms/bfd/model/rif/samples/StaticRifResource.java

```diff
@@ -127,206 +150,229 @@ public enum StaticRifResource {
       309),
-
+  /** Synthetic Inpatient 2014_2016 Remote S3 data. */
   SYNTHETIC_INPATIENT_2014_2016(
       remoteS3Data(TestDataSetLocation.SYNTHETIC_DATA, "synthetic-inpatient-2014-2016.rif"),
       RifFileType.INPATIENT,
       387),
-
+  /** Synthetic PDE 2014 Remote S3 data. */
   SYNTHETIC_PDE_2014(
       remoteS3Data(TestDataSetLocation.SYNTHETIC_DATA, "synthetic-pde-2014.rif"),
       RifFileType.PDE,
       127643),
-
+  /** Synthetic PDE 2015 Remote S3 data. */
   SYNTHETIC_PDE_2015(
       remoteS3Data(TestDataSetLocation.SYNTHETIC_DATA, "synthetic-pde-2015.rif"),
       RifFileType.PDE,
       140176),
-
+  /** Synthetic PDE 2016 Remote S3 data. */
   SYNTHETIC_PDE_2016(
       remoteS3Data(TestDataSetLocation.SYNTHETIC_DATA, "synthetic-pde-2016.rif"),
       RifFileType.PDE,
       145526),
-
+  /** Synthetic Outpatient 1999_1999 Remote S3 data. */
   SYNTHETIC_OUTPATIENT_1999_1999(
       remoteS3Data(TestDataSetLocation.SYNTHETIC_DATA, "synthetic-outpatient-1999-1999.rif"),
       RifFileType.OUTPATIENT,
       20744),
-
+  /** Synthetic Outpatient 2000_1999 Remote S3 data. */
   SYNTHETIC_OUTPATIENT_2000_1999(
       remoteS3Data(TestDataSetLocation.SYNTHETIC_DATA, "synthetic-outpatient-2000-1999.rif"),
       RifFileType.OUTPATIENT,
       22439),
-
+  /** Synthetic Outpatient 2001_1999 Remote S3 data. */
   SYNTHETIC_OUTPATIENT_2001_1999(
       remoteS3Data(TestDataSetLocation.SYNTHETIC_DATA, "synthetic-outpatient-2001-1999.rif"),
       RifFileType.OUTPATIENT,
       23241),
-
+  /** Synthetic Outpatient 2002_2000 Remote S3 data. */
   SYNTHETIC_OUTPATIENT_2002_2000(
       remoteS3Data(TestDataSetLocation.SYNTHETIC_DATA, "synthetic-outpatient-2002-2000.rif"),
       RifFileType.OUTPATIENT,
       24575),
-
+  /** Synthetic Outpatient 2014_2014 Remote S3 data. */
   SYNTHETIC_OUTPATIENT_2014_2014(
       remoteS3Data(TestDataSetLocation.SYNTHETIC_DATA, "synthetic-outpatient-2014-2014.rif"),
       RifFileType.OUTPATIENT,
       25194),
-
+  /** Synthetic Outpatient 2015_2014 Remote S3 data. */
   SYNTHETIC_OUTPATIENT_2015_2014(
       remoteS3Data(TestDataSetLocation.SYNTHETIC_DATA, "synthetic-outpatient-2015-2014.rif"),
       RifFileType.OUTPATIENT,
       26996),
-
+  /** Synthetic Outpatient 2016_2014 Remote S3 data. */
   SYNTHETIC_OUTPATIENT_2016_2014(
       remoteS3Data(TestDataSetLocation.SYNTHETIC_DATA, "synthetic-outpatient-2016-2014.rif"),
       RifFileType.OUTPATIENT,
       27955),
-
+  /** Synthetic Hicn Multiple Benes S3 data. */
   SAMPLE_HICN_MULT_BENES(
       resourceUrl("rif-static-samples/sample-hicn-mult-bene-beneficiaries.txt"),
       RifFileType.BENEFICIARY,
       10),
-
+  /** Synthetic Hicn Multiple Beneficiary History S3 data. */
   SAMPLE_HICN_MULT_BENES_BENEFICIARY_HISTORY(
       resourceUrl("rif-static-samples/sample-hicn-mult-bene-beneficiaryhistory.txt"),
       RifFileType.BENEFICIARY_HISTORY,
       7);
 
+  /** {@link Supplier} the Resource URL Supplier. */
```

Comment:
Don't want or need a link to the type here; that information is automatically defined in the docs and is visible below.

Reply:
done